packages/ncids-js/src/utils/focus-trap.ts

```diff
@@ -22,11 +20,23 @@ export class FocusTrap {
 	 *
 	 * @param {HTMLElement} element Component being created.
 	 */
-	public constructor(element: HTMLElement) {
+	public constructor(element: HTMLElement, options?: string) {
```

Comment:
this should not be like that. You should never have something called "options" that is just a string, and a specific option at that.

We should track this as a backlog item to be fixed after this release. (Also, this should be using destructuring with a default...)

e.g.,
```
public constructor(element: HTMLElement, {
  focusableElementSelector = 'button, [href], input, select, textarea, [tabindex]:not([tabindex="-1"])'
})
```